### PR TITLE
Add permissive robots.txt

### LINKED ISSUE #470: Add a default robots.txt to /public
Just an idea. I could PR a permissive one really fast.

### PR DESCRIPTION
**Pull request by watzon**
Closes #470

### CODE REVIEW

File: src/browser_app_skeleton/public/robots.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Allow: *
```
Comment (paulcsmith) [inline src/browser_app_skeleton/public/robots.txt]: I've done a bit of research and it looks like `Disallow` is better supported

```suggestion
Disallow:
```

Thoughts on changing to that?

File: src/browser_app_skeleton/public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
```
Comment (jwoertink) [inline src/browser_app_skeleton/public/robots.txt]: Can you add a comment just above this pointing to https://www.robotstxt.org/robotstxt.html ? Just a quick little reference saying "For info on updating this" or whatever.

File: src/browser_app_skeleton/public/robots.txt (hunk 1 above)
Comment (wontruefree) [reply src/browser_app_skeleton/public/robots.txt]: That seems confusing.  Maybe we can add a comment.

Comment (paulcsmith) [reply src/browser_app_skeleton/public/robots.txt]: Yeah it is super weird. Comment sounds good 👍

Comment (watzon) [reply src/browser_app_skeleton/public/robots.txt]: Yeah really it shouldn't need either afaik, but changing to disallow could work fine

File: src/browser_app_skeleton/public/robots.txt (hunk 2 above)
Comment (watzon) [reply src/browser_app_skeleton/public/robots.txt]: Done :+1:

**Comment by paulcsmith** [comment]
Thank you!

**Pull request merged**